q2_feature_table/plugin_setup.py

```diff
@@ -300,6 +300,41 @@
                 "on https://docs.qiime2.org for additional details."
 )
 
+plugin.methods.register_function(
+    function=q2_feature_table.filter_seqs,
+    inputs={
+        'data': FeatureData[Sequence],
+    },
+    parameters={
+        'metadata': Metadata,
+        'where': Str,
+        'exclude_ids': Bool
+    },
+    outputs=[('filtered_data', FeatureData[Sequence])],
+    input_descriptions={
+        'data': 'The sequences from which features should be filtered.',
+    },
+    parameter_descriptions={
+        'metadata': 'Feature metadata used for id-based filtering, with '
+                    '`where` parameter when selecting features to retain, or '
+                    'with `exclude_ids` when selecting features to discard.',
+        'where': 'SQLite WHERE clause specifying feature metadata criteria '
+                 'that must be met to be included in the filtered feature '
+                 'table. If not provided, all features in `metadata` that are '
+                 'also in the sequences will be retained.',
+        'exclude_ids': 'If `True`, the features selected by `metadata` or '
```

Comment:
`True` depends on the interface, I would go with just "true"

Reply:
👍 This was copied from the other `filter` methods in this plugin. Want me to make this change in all three locations?

Reply:
That would be awesome!